cadl-tests/.vscode/launch.json

```diff
@@ -12,6 +12,16 @@
                 "<node_internals>/**"
             ],
             "type": "node"
+        },
```

Comment:
This is for vs code debugging configuration. With this, you can just click the debug button to debug cadl codegen.

Reply:
so far it is fine.

we might see later if `"runtimeArgs": ["cadl", "compile", "builtin.cadl"],` will affect other usages.

Reply:
You can modify this `runtimeArgs` to debug other cadl files generation

Reply:
yes, but I don't want to either commit it every time it get modified locally, or take the pain to exclude it from commit.

we might later see if put it to ignore.